pwa/src/templates/componentDetail/ComponentsDetailTemplate.tsx

```diff
@@ -94,42 +98,49 @@ export const ComponentsDetailTemplate: React.FC<ComponentsDetailTemplateProps> =
                         <Link icon={<ArrowRightIcon />} iconAlign="start">
                           {_getComponent.data.embedded?.legal.embedded?.repoOwner.name}
                         </Link>
+                        .
                       </span>
                     </div>
                   ) : (
-                    "Er is geen informatie beschikbaar"
+                    "Er is geen informatie beschikbaar."
```

Comment:
I'd opt out of using a ternary expression here.

Explicit expressions are easier to read, so in this case:

```
_getComponent.data.embedded?.legal.embedded?.repoOwner.name && <>...</>
!_getComponent.data.embedded?.legal.embedded?.repoOwner.name && <>...</>
```

Reply:
Also, for readability it might be worth it to save this within `state`. That way, we could have smaller variable names.

(dummy code)

```
const embedded = _getComponent.data.embedded;

// or even further

const legal = embedded.legal.embedded;
```